src/implementations/twig/components/button/button.story.js

```diff
@@ -146,7 +146,7 @@ const prepareData = (data, args) => {
   if (args.icon_name && args.icon_name !== 'none') {
     data.icon = {};
     data.icon.name = args.icon_name;
-    data.icon.size = system === 'eu' ? 's' : 'xs';
+    data.icon.size = 'xs';
```

Comment:
Can we make the hide_label control conditional? It should appear only when an icon has been chosen

Reply:
Well, can we do that? :)
As far as I understood, the conditional control (with 'if') only work with boolean, I don't know if we can detect if a select list has a specific value

Reply:
I updated the story to set a default icon if the hide_label control is true.
That's not perfect, but already better

Reply:
but we have already in the button story file the same condition used for the icon args:
`if: { arg: 'icon_name', neq: 'none' },`

Reply:
that's true... I'll update